Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/GUIMainWindow.java

```diff
@@ -0,0 +1,203 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.awt.ComponentOrientation;
+import java.awt.Dimension;
+import java.awt.EventQueue;
+import java.awt.Font;
+
+import javax.swing.BoxLayout;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JSeparator;
+
+public class GUIMainWindow implements UserInterface{
+	Game game;
+	boolean firstRun=true;
+	
+	JLabel lblName;
+	JLabel lblGamesWon;
+	JLabel lblLettersTried;
+	JLabel openedWord;
+	JLabel lblStatus;
+	GUIGamePanel hangmanPanel;
+
+	public GUIMainWindow (Game newGame){
```

Comment:
I think passing around the `Game` object leads to more coupling than is necessary between the GUI and `Game`.  An alternative could be for the `Game` object to provide the necessary information when it calls the GUI method, for example passing the hangman stage to `wrongGuess`